docs/install_guides/installing-openvino-apt.md

```diff
@@ -30,29 +30,35 @@ The complete list of supported hardware is available in the [Release Notes](http
       sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
       ```
       > **NOTE**: You might need to install GnuPG: `sudo apt-get install gnupg`   
-      
+
 2.	Add the repository via the following command:
 
-   * On Ubuntu 18
-   ```sh
-   echo "deb https://apt.repos.intel.com/openvino/2022 bionic main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list
-   ```
-   
-   * On Ubuntu 20
-   ```sh
-   echo "deb https://apt.repos.intel.com/openvino/2022 focal main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list
-   ```  
+  @sphinxdirective
+
+  .. tab:: On Ubuntu 18
+
+     .. code-block:: sh
+
+        echo "deb https://apt.repos.intel.com/openvino/2022 bionic main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list
+
+  .. tab:: On Ubuntu 20
+
+     .. code-block:: sh
+
+        echo "deb https://apt.repos.intel.com/openvino/2022 focal main" | sudo tee /etc/apt/sources.list.d/intel-openvino-2022.list
+
+  @endsphinxdirective
```

Comment:
I just have a little concern about the output. As I remember, I tried this and there were some issues with the indentation or something so I reverted it back. But we will see this time. :)

Reply:
IMHO, previous version looks more accurate, than
![image](https://user-images.githubusercontent.com/3531525/158337686-29e789b6-725c-4ed8-9fba-b83b34936ed4.png)
It had better to use 'Ubuntu 18' instead 'On Ubuntu 18' for tabs.

Reply:
fixed now